discovery-provider/src/queries/get_playlist_tracks.py

```diff
@@ -0,0 +1,63 @@
+import logging # pylint: disable=C0302
+import sqlalchemy
+
+from src.models import Playlist, Track
+from src.utils import helpers
+from src.utils.db_session import get_db_read_replica
+from src.queries.query_helpers import get_current_user_id, get_pagination_vars, \
+  populate_track_metadata, add_users_to_tracks
+
+logger = logging.getLogger(__name__)
+
+def get_playlist_tracks(args):
+    playlists = []
+    current_user_id = get_current_user_id(required=False)
+
+    db = get_db_read_replica()
+    with db.scoped_session() as session:
+        try:
+            playlist_id = args.get("playlist_id")
+            playlist = (
+                session
+                    .query(Playlist)
+                    .filter(
+                        Playlist.is_current == True,
+                        Playlist.playlist_id == playlist_id
+                    )
+                    .first()
+            )
+            if playlist == None:
+                return None
+
+            playlist_track_ids = [ track_id['track'] for track_id in playlist.playlist_contents['track_ids']]
+            (limit, offset) = get_pagination_vars()
+            query_playlist_track_ids = playlist_track_ids[offset:offset+limit] 
+            
+            playlist_tracks = (
+                session
+                    .query(Track)
+                    .filter(
+                        Track.is_current == True,
+                        Track.track_id.in_(query_playlist_track_ids)
+                    )
+                    .all()
+            )
+
+            tracks = helpers.query_result_to_list(playlist_tracks)
+            tracks = populate_track_metadata(
+                session, playlist_track_ids, tracks, current_user_id)
+
+            if "with_users" in args and args.get("with_users") != 'false':
```

Comment:
can we add this 'false' string check to the list of polish items? we should really be passing booleans in as Booleans...